com.unity.build-report-inspector/Editor/BuildReportInspector/Mobile/Tools/Utilities.cs

```diff
@@ -0,0 +1,36 @@
+#if UNITY_EDITOR
```

Comment:
No need for #if as this script is under `Editor` folder.

Reply:
Fixed